src/plugin/types/Element.ts

```diff
@@ -32,7 +32,7 @@ export type FieldDescription =
   | TextFieldDescription
   | RichTextFieldDescription
   | CheckboxFieldDescription
-  | CustomFieldDescription
+  | CustomFieldDescription<unknown, unknown>
```

Comment:
This line is producing a linter error, but the checks haven't run for this PR.

Reply:
This is not needed any more – we can remove.